R/pkg/R/sparkR.R

```diff
@@ -624,8 +624,8 @@ sparkConfToSubmitOps[["spark.driver.extraClassPath"]]   <- "--driver-class-path"
 sparkConfToSubmitOps[["spark.driver.extraJavaOptions"]] <- "--driver-java-options"
 sparkConfToSubmitOps[["spark.driver.extraLibraryPath"]] <- "--driver-library-path"
 sparkConfToSubmitOps[["spark.master"]] <- "--master"
-sparkConfToSubmitOps[["spark.yarn.keytab"]] <- "--keytab"
```

Comment:
@felixcheung 

This seems wrong, since existing apps are using the old config, not the new one, so you may need some backwards compatibility code.

Reply:
Added backwards compatibility